docs/community-clients/index.asciidoc

```diff
@@ -1,223 +1,230 @@
-=== Java
+See the {client}/java-api/current/index.html[official Elasticsearch Java client].
 
 * https://github.com/searchbox-io/Jest[Jest]:
   Java Rest client.
-* There is of course the {client}/java-api/current/index.html[native ES Java client]
 
-[[community-javascript]]
-=== JavaScript
+[[javascript]]
+== JavaScript
 
 See the {client}/javascript-api/current/index.html[official Elasticsearch JavaScript client].
 
 * https://github.com/fullscale/elastic.js[Elastic.js]:
   A JavaScript implementation of the Elasticsearch Query DSL and Core API.
 
-* https://github.com/phillro/node-elasticsearch-client[node-elasticsearch-client]:
-  A NodeJS client for Elasticsearch.
+* https://github.com/printercu/elastics[elastics]: Simple tiny client that just works
 
-* https://github.com/ramv/node-elastical[node-elastical]:
-  Node.js client for the Elasticsearch REST API
+* https://github.com/roundscope/ember-data-elasticsearch-kit[ember-data-elasticsearch-kit]:
+  An ember-data kit for both pushing and querying objects to Elasticsearch cluster
 
-* https://github.com/printercu/elastics[elastics]: Simple tiny client that just works
+The following project appears to be abandoned:
 
-[[community-groovy]]
-=== Groovy
+* https://github.com/ramv/node-elastical[node-elastical]:
+  Node.js client for the Elasticsearch REST API
 
-See the {client}/groovy-api/current/index.html[official Elasticsearch Groovy client]
 
-[[community-dotnet]]
-=== .NET
+[[dotnet]]
+== .NET
 
 See the {client}/net-api/current/index.html[official Elasticsearch .NET client].
 
 * https://github.com/Yegoroff/PlainElastic.Net[PlainElastic.Net]:
   .NET client.
 
-* https://github.com/medcl/ElasticSearch.Net[ElasticSearch.NET]:
-  .NET client.
+[[ocaml]]
+== OCaml
 
+The following project appears to be abandoned:
 
-[[community-haskell]]
-=== Haskell
-* https://github.com/bitemyapp/bloodhound[bloodhound]:
-  Haskell client and DSL.
+* https://github.com/tovbinm/ocaml-elasticsearch[ocaml-elasticsearch]:
+  OCaml client for Elasticsearch
 
+[[perl]]
+== Perl
 
-[[community-scala]]
-=== Scala
+See the {client}/perl-api/current/index.html[official Elasticsearch Perl client].
 
-* https://github.com/sksamuel/elastic4s[elastic4s]:
-  Scala DSL.
+* https://metacpan.org/pod/Elastijk[Elastijk]: A low level minimal HTTP client.
 
-* https://github.com/scalastuff/esclient[esclient]:
-  Thin Scala client.
 
-* https://github.com/bsadeh/scalastic[scalastic]:
-  Scala client.
+[[php]]
+== PHP
 
-* https://github.com/gphat/wabisabi[wabisabi]:
-  Asynchronous REST API Scala client.
+See the {client}/php-api/current/index.html[official Elasticsearch PHP client].
 
+* http://github.com/ruflin/Elastica[Elastica]:
+  PHP client.
 
-[[community-clojure]]
-=== Clojure
+* http://github.com/nervetattoo/elasticsearch[elasticsearch] PHP client.
 
-* http://github.com/clojurewerkz/elastisch[Elastisch]:
-  Clojure client.
+* https://github.com/nervetattoo/elasticsearch[elasticsearch]
+  PHP 5.3 client
```

Comment:
Is this a PHP 5.3 port of the official Elasticsearch client? Elastica is 5.3 compatible - WMF ran in on 5.3.